src/ralph/lib/transitions/models.py

```diff
@@ -334,10 +338,6 @@ def __new__(cls, name, bases, attrs):
             _transitions_fields[new_class] = fields
             for field in fields:
                 new_class.add_to_class(
-                    'run_{}_transition'.format(field),
```

Comment:
@ar4s What do you think about delete this? and use only from run_field_transition()